From: Sam Hartman <hartmans@debian.org>
Date: Mon, 11 Sep 2023 14:00:42 -0600
Subject: PAM-manpage-section

Patch to put the PAM manpage in section 7 (general topics) instead of 8
(system administration commands)

Authors: Steve Langasek <vorlon@debian.org>

Upstream status: maybe provide a backwards-compatibility link first?
---
 doc/man/meson.build                             |   2 +-
 doc/man/misc_conv.3.xml                         |   2 +-
 doc/man/pam.7.xml                               | 212 ++++++++++++++++++++++++
 doc/man/pam.8.xml                               | 212 ------------------------
 doc/man/pam_acct_mgmt.3.xml                     |   2 +-
 doc/man/pam_authenticate.3.xml                  |   2 +-
 doc/man/pam_chauthtok.3.xml                     |   2 +-
 doc/man/pam_conv.3.xml                          |   2 +-
 doc/man/pam_error.3.xml                         |   2 +-
 doc/man/pam_get_authtok.3.xml                   |   2 +-
 doc/man/pam_getenv.3.xml                        |   2 +-
 doc/man/pam_getenvlist.3.xml                    |   2 +-
 doc/man/pam_info.3.xml                          |   2 +-
 doc/man/pam_misc_drop_env.3.xml                 |   2 +-
 doc/man/pam_misc_paste_env.3.xml                |   2 +-
 doc/man/pam_misc_setenv.3.xml                   |   2 +-
 doc/man/pam_prompt.3.xml                        |   2 +-
 doc/man/pam_putenv.3.xml                        |   2 +-
 doc/man/pam_strerror.3.xml                      |   2 +-
 doc/man/pam_syslog.3.xml                        |   2 +-
 modules/pam_access/access.conf.5.xml            |   2 +-
 modules/pam_access/pam_access.8.xml             |   2 +-
 modules/pam_debug/pam_debug.8.xml               |   2 +-
 modules/pam_deny/pam_deny.8.xml                 |   2 +-
 modules/pam_echo/pam_echo.8.xml                 |   2 +-
 modules/pam_env/pam_env.8.xml                   |   2 +-
 modules/pam_env/pam_env.conf.5.xml              |   2 +-
 modules/pam_exec/pam_exec.8.xml                 |   2 +-
 modules/pam_faildelay/pam_faildelay.8.xml       |   2 +-
 modules/pam_filter/pam_filter.8.xml             |   2 +-
 modules/pam_ftp/pam_ftp.8.xml                   |   2 +-
 modules/pam_group/group.conf.5.xml              |   2 +-
 modules/pam_group/pam_group.8.xml               |   2 +-
 modules/pam_issue/pam_issue.8.xml               |   2 +-
 modules/pam_keyinit/pam_keyinit.8.xml           |   2 +-
 modules/pam_lastlog/pam_lastlog.8.xml           |   2 +-
 modules/pam_limits/limits.conf.5.xml            |   2 +-
 modules/pam_limits/pam_limits.8.xml             |   2 +-
 modules/pam_listfile/pam_listfile.8.xml         |   2 +-
 modules/pam_localuser/pam_localuser.8.xml       |   2 +-
 modules/pam_loginuid/pam_loginuid.8.xml         |   2 +-
 modules/pam_mail/pam_mail.8.xml                 |   2 +-
 modules/pam_mkhomedir/pam_mkhomedir.8.xml       |   2 +-
 modules/pam_motd/pam_motd.8.xml                 |   2 +-
 modules/pam_namespace/namespace.conf.5.xml      |   2 +-
 modules/pam_namespace/pam_namespace.8.xml       |   2 +-
 modules/pam_nologin/pam_nologin.8.xml           |   2 +-
 modules/pam_permit/pam_permit.8.xml             |   2 +-
 modules/pam_pwhistory/pam_pwhistory.8.xml       |   2 +-
 modules/pam_rhosts/pam_rhosts.8.xml             |   2 +-
 modules/pam_rootok/pam_rootok.8.xml             |   2 +-
 modules/pam_securetty/pam_securetty.8.xml       |   2 +-
 modules/pam_selinux/pam_selinux.8.xml           |   2 +-
 modules/pam_sepermit/pam_sepermit.8.xml         |   2 +-
 modules/pam_sepermit/sepermit.conf.5.xml        |   2 +-
 modules/pam_shells/pam_shells.8.xml             |   2 +-
 modules/pam_succeed_if/pam_succeed_if.8.xml     |   2 +-
 modules/pam_time/pam_time.8.xml                 |   2 +-
 modules/pam_time/time.conf.5.xml                |   2 +-
 modules/pam_timestamp/pam_timestamp.8.xml       |   2 +-
 modules/pam_timestamp/pam_timestamp_check.8.xml |   2 +-
 modules/pam_tty_audit/pam_tty_audit.8.xml       |   2 +-
 modules/pam_umask/pam_umask.8.xml               |   2 +-
 modules/pam_unix/pam_unix.8.xml                 |   2 +-
 modules/pam_userdb/pam_userdb.8.xml             |   2 +-
 modules/pam_warn/pam_warn.8.xml                 |   2 +-
 modules/pam_wheel/pam_wheel.8.xml               |   2 +-
 modules/pam_xauth/pam_xauth.8.xml               |   2 +-
 68 files changed, 278 insertions(+), 278 deletions(-)
 create mode 100644 doc/man/pam.7.xml
 delete mode 100644 doc/man/pam.8.xml

diff --git a/doc/man/meson.build b/doc/man/meson.build
index b90b157..71650ec 100644
--- a/doc/man/meson.build
+++ b/doc/man/meson.build
@@ -35,7 +35,7 @@ foreach man: [['misc_conv.3', []],
               ['pam_syslog.3', ['pam_vsyslog.3']],
               ['pam_xauth_data.3', []],
               ['pam.conf.5', ['pam.d.5']],
-              ['pam.8', ['PAM.8']],
+              ['pam.7', ['PAM.7']],
              ]
   xml = man[0] + '.xml'
 
diff --git a/doc/man/misc_conv.3.xml b/doc/man/misc_conv.3.xml
index 92d4acd..2971b3a 100644
--- a/doc/man/misc_conv.3.xml
+++ b/doc/man/misc_conv.3.xml
@@ -168,7 +168,7 @@
         <refentrytitle>pam_conv</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam.7.xml b/doc/man/pam.7.xml
new file mode 100644
index 0000000..cb6a7d8
--- /dev/null
+++ b/doc/man/pam.7.xml
@@ -0,0 +1,212 @@
+<refentry xmlns="http://docbook.org/ns/docbook" version="5.0" xml:id="pam8">
+
+  <refmeta>
+    <refentrytitle>pam</refentrytitle>
+    <manvolnum>7</manvolnum>
+    <refmiscinfo class="source">Linux-PAM</refmiscinfo>
+    <refmiscinfo class="manual">Linux-PAM Manual</refmiscinfo>
+  </refmeta>
+
+  <refnamediv xml:id="pam8-name">
+    <refname>PAM</refname>
+    <refname>pam</refname>
+    <refpurpose>Pluggable Authentication Modules for Linux</refpurpose>
+  </refnamediv>
+
+  <refsect1 xml:id="pam8-description">
+    <title>DESCRIPTION</title>
+    <para>
+      This manual is intended to offer a quick introduction to
+      <emphasis remap="B">Linux-PAM</emphasis>. For more information
+      the reader is directed to the
+      <emphasis remap="B">Linux-PAM system administrators' guide</emphasis>.
+    </para>
+
+    <para>
+      <emphasis remap="B">Linux-PAM</emphasis> is a system of libraries
+      that handle the authentication tasks of applications (services) on
+      the system. The library provides a stable general interface
+      (Application Programming Interface - API) that privilege granting
+      programs (such as <citerefentry>
+      <refentrytitle>login</refentrytitle><manvolnum>1</manvolnum>
+      </citerefentry> and <citerefentry>
+      <refentrytitle>su</refentrytitle><manvolnum>1</manvolnum>
+      </citerefentry>) defer to to perform standard authentication tasks.
+    </para>
+
+    <para>
+      The principal feature of the PAM approach is that the nature of the
+      authentication is dynamically configurable.  In other words, the
+      system administrator is free to choose how individual
+      service-providing applications will authenticate users. This dynamic
+      configuration is set by the contents of the single
+      <emphasis remap="B">Linux-PAM</emphasis> configuration file
+      <filename>/etc/pam.conf</filename>. Alternatively and preferably,
+      the configuration can be set by individual configuration files
+      located in a <filename>pam.d</filename> directory. The presence of this
+      directory will cause <emphasis remap="B">Linux-PAM</emphasis> to
+      <emphasis remap="I">ignore</emphasis> <filename>/etc/pam.conf</filename>.
+    </para>
+
+    <para>
+      Vendor-supplied PAM configuration files might be installed in
+      the system directory <filename>/usr/lib/pam.d/</filename> or
+      a configurable vendor specific directory instead
+      of the machine configuration directory <filename>/etc/pam.d/</filename>.
+      If no machine configuration file is found, the vendor-supplied file
+      is used. All files in <filename>/etc/pam.d/</filename> override
+      files with the same name in other directories.
+    </para>
+
+<para>From the point of view of the system administrator, for whom this
+manual is provided, it is not of primary importance to understand the
+internal behavior of the
+<emphasis remap="B">Linux-PAM</emphasis>
+library.  The important point to recognize is that the configuration
+file(s)
+<emphasis remap="I">define</emphasis>
+the connection between applications
+<emphasis remap="B"/>(<emphasis remap="B">services</emphasis>)
+and the pluggable authentication modules
+<emphasis remap="B"/>(<emphasis remap="B">PAM</emphasis>s)
+that perform the actual authentication tasks.</para>
+
+
+<para><emphasis remap="B">Linux-PAM</emphasis>
+separates the tasks of
+<emphasis remap="I">authentication</emphasis>
+into four independent management groups:
+<emphasis remap="B">account</emphasis> management;
+<emphasis remap="B">auth</emphasis>entication management;
+<emphasis remap="B">password</emphasis> management;
+and
+<emphasis remap="B">session</emphasis> management.
+(We highlight the abbreviations used for these groups in the
+configuration file.)</para>
+
+
+<para>Simply put, these groups take care of different aspects of a typical
+user's request for a restricted service:</para>
+
+
+<para><emphasis remap="B">account</emphasis> -
+provide account verification types of service: has the user's password
+expired?; is this user permitted access to the requested service?</para>
+
+<!-- .br -->
+<para><emphasis remap="B">auth</emphasis>entication -
+authenticate a user and set up user credentials. Typically this is via
+some challenge-response request that the user must satisfy: if you are
+who you claim to be please enter your password. Not all authentications
+are of this type, there exist hardware based authentication schemes
+(such as the use of smart-cards and biometric devices), with suitable
+modules, these may be substituted seamlessly for more standard
+approaches to authentication - such is the flexibility of
+<emphasis remap="B">Linux-PAM</emphasis>.</para>
+
+<!-- .br -->
+<para><emphasis remap="B">password</emphasis> -
+this group's responsibility is the task of updating authentication
+mechanisms. Typically, such services are strongly coupled to those of
+the
+<emphasis remap="B">auth</emphasis>
+group. Some authentication mechanisms lend themselves well to being
+updated with such a function. Standard UN*X password-based access is
+the obvious example: please enter a replacement password.</para>
+
+<!-- .br -->
+<para><emphasis remap="B">session</emphasis> -
+this group of tasks cover things that should be done prior to a
+service being given and after it is withdrawn. Such tasks include the
+maintenance of audit trails and the mounting of the user's home
+directory. The
+<emphasis remap="B">session</emphasis>
+management group is important as it provides both an opening and
+closing hook for modules to affect the services available to a user.</para>
+
+</refsect1>
+
+  <refsect1 xml:id="pam8-files">
+    <title>FILES</title>
+    <variablelist>
+      <varlistentry>
+        <term>/etc/pam.conf</term>
+        <listitem>
+          <para>the configuration file</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>/etc/pam.d</term>
+        <listitem>
+          <para>
+            the <emphasis remap="B">Linux-PAM</emphasis> configuration
+            directory. Generally, if this directory is present, the
+            <filename>/etc/pam.conf</filename> file is ignored.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>/usr/lib/pam.d</term>
+        <listitem>
+          <para>
+            the <emphasis remap="B">Linux-PAM</emphasis> vendor configuration
+            directory. Files in <filename>/etc/pam.d</filename> override
+            files with the same name in this directory.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry condition="with_vendordir">
+        <term>%vendordir%/pam.d</term>
+        <listitem>
+          <para>
+	    additional <emphasis remap="B">Linux-PAM</emphasis> vendor
+	    configuration directory. Files in <filename>/etc/pam.d</filename>
+	    and <filename>/usr/lib/pam.d</filename> override files with the
+	    same name in this directory.
+          </para>
+        </listitem>
+      </varlistentry>
+    </variablelist>
+  </refsect1>
+
+  <refsect1 xml:id="pam8-errors">
+    <title>ERRORS</title>
+    <para>
+      Typically errors generated by the
+      <emphasis remap="B">Linux-PAM</emphasis> system of libraries, will
+      be written to <citerefentry>
+      <refentrytitle>syslog</refentrytitle><manvolnum>3</manvolnum>
+      </citerefentry>.
+    </para>
+  </refsect1>
+
+  <refsect1 xml:id="pam8-conforming_to">
+    <title>CONFORMING TO</title>
+    <para>
+      DCE-RFC 86.0, October 1995.
+      Contains additional features, but remains backwardly compatible
+      with this RFC.
+    </para>
+  </refsect1>
+
+  <refsect1 xml:id="pam8-see_also">
+    <title>SEE ALSO</title>
+    <para>
+      <citerefentry>
+        <refentrytitle>pam</refentrytitle><manvolnum>3</manvolnum>
+      </citerefentry>,
+      <citerefentry>
+        <refentrytitle>pam_authenticate</refentrytitle><manvolnum>3</manvolnum>
+      </citerefentry>,
+      <citerefentry>
+        <refentrytitle>pam_sm_setcred</refentrytitle><manvolnum>3</manvolnum>
+      </citerefentry>,
+      <citerefentry>
+        <refentrytitle>pam_strerror</refentrytitle><manvolnum>3</manvolnum>
+      </citerefentry>,
+      <citerefentry>
+        <refentrytitle>PAM</refentrytitle><manvolnum>8</manvolnum>
+      </citerefentry>
+    </para>
+  </refsect1>
+</refentry>
diff --git a/doc/man/pam.8.xml b/doc/man/pam.8.xml
deleted file mode 100644
index 7f3b051..0000000
--- a/doc/man/pam.8.xml
+++ /dev/null
@@ -1,212 +0,0 @@
-<refentry xmlns="http://docbook.org/ns/docbook" version="5.0" xml:id="pam8">
-
-  <refmeta>
-    <refentrytitle>pam</refentrytitle>
-    <manvolnum>8</manvolnum>
-    <refmiscinfo class="source">Linux-PAM</refmiscinfo>
-    <refmiscinfo class="manual">Linux-PAM Manual</refmiscinfo>
-  </refmeta>
-
-  <refnamediv xml:id="pam8-name">
-    <refname>PAM</refname>
-    <refname>pam</refname>
-    <refpurpose>Pluggable Authentication Modules for Linux</refpurpose>
-  </refnamediv>
-
-  <refsect1 xml:id="pam8-description">
-    <title>DESCRIPTION</title>
-    <para>
-      This manual is intended to offer a quick introduction to
-      <emphasis remap="B">Linux-PAM</emphasis>. For more information
-      the reader is directed to the
-      <emphasis remap="B">Linux-PAM system administrators' guide</emphasis>.
-    </para>
-
-    <para>
-      <emphasis remap="B">Linux-PAM</emphasis> is a system of libraries
-      that handle the authentication tasks of applications (services) on
-      the system. The library provides a stable general interface
-      (Application Programming Interface - API) that privilege granting
-      programs (such as <citerefentry>
-      <refentrytitle>login</refentrytitle><manvolnum>1</manvolnum>
-      </citerefentry> and <citerefentry>
-      <refentrytitle>su</refentrytitle><manvolnum>1</manvolnum>
-      </citerefentry>) defer to to perform standard authentication tasks.
-    </para>
-
-    <para>
-      The principal feature of the PAM approach is that the nature of the
-      authentication is dynamically configurable.  In other words, the
-      system administrator is free to choose how individual
-      service-providing applications will authenticate users. This dynamic
-      configuration is set by the contents of the single
-      <emphasis remap="B">Linux-PAM</emphasis> configuration file
-      <filename>/etc/pam.conf</filename>. Alternatively and preferably,
-      the configuration can be set by individual configuration files
-      located in a <filename>pam.d</filename> directory. The presence of this
-      directory will cause <emphasis remap="B">Linux-PAM</emphasis> to
-      <emphasis remap="I">ignore</emphasis> <filename>/etc/pam.conf</filename>.
-    </para>
-
-    <para>
-      Vendor-supplied PAM configuration files might be installed in
-      the system directory <filename>/usr/lib/pam.d/</filename> or
-      a configurable vendor specific directory instead
-      of the machine configuration directory <filename>/etc/pam.d/</filename>.
-      If no machine configuration file is found, the vendor-supplied file
-      is used. All files in <filename>/etc/pam.d/</filename> override
-      files with the same name in other directories.
-    </para>
-
-<para>From the point of view of the system administrator, for whom this
-manual is provided, it is not of primary importance to understand the
-internal behavior of the
-<emphasis remap="B">Linux-PAM</emphasis>
-library.  The important point to recognize is that the configuration
-file(s)
-<emphasis remap="I">define</emphasis>
-the connection between applications
-<emphasis remap="B"/>(<emphasis remap="B">services</emphasis>)
-and the pluggable authentication modules
-<emphasis remap="B"/>(<emphasis remap="B">PAM</emphasis>s)
-that perform the actual authentication tasks.</para>
-
-
-<para><emphasis remap="B">Linux-PAM</emphasis>
-separates the tasks of
-<emphasis remap="I">authentication</emphasis>
-into four independent management groups:
-<emphasis remap="B">account</emphasis> management;
-<emphasis remap="B">auth</emphasis>entication management;
-<emphasis remap="B">password</emphasis> management;
-and
-<emphasis remap="B">session</emphasis> management.
-(We highlight the abbreviations used for these groups in the
-configuration file.)</para>
-
-
-<para>Simply put, these groups take care of different aspects of a typical
-user's request for a restricted service:</para>
-
-
-<para><emphasis remap="B">account</emphasis> -
-provide account verification types of service: has the user's password
-expired?; is this user permitted access to the requested service?</para>
-
-<!-- .br -->
-<para><emphasis remap="B">auth</emphasis>entication -
-authenticate a user and set up user credentials. Typically this is via
-some challenge-response request that the user must satisfy: if you are
-who you claim to be please enter your password. Not all authentications
-are of this type, there exist hardware based authentication schemes
-(such as the use of smart-cards and biometric devices), with suitable
-modules, these may be substituted seamlessly for more standard
-approaches to authentication - such is the flexibility of
-<emphasis remap="B">Linux-PAM</emphasis>.</para>
-
-<!-- .br -->
-<para><emphasis remap="B">password</emphasis> -
-this group's responsibility is the task of updating authentication
-mechanisms. Typically, such services are strongly coupled to those of
-the
-<emphasis remap="B">auth</emphasis>
-group. Some authentication mechanisms lend themselves well to being
-updated with such a function. Standard UN*X password-based access is
-the obvious example: please enter a replacement password.</para>
-
-<!-- .br -->
-<para><emphasis remap="B">session</emphasis> -
-this group of tasks cover things that should be done prior to a
-service being given and after it is withdrawn. Such tasks include the
-maintenance of audit trails and the mounting of the user's home
-directory. The
-<emphasis remap="B">session</emphasis>
-management group is important as it provides both an opening and
-closing hook for modules to affect the services available to a user.</para>
-
-</refsect1>
-
-  <refsect1 xml:id="pam8-files">
-    <title>FILES</title>
-    <variablelist>
-      <varlistentry>
-        <term>/etc/pam.conf</term>
-        <listitem>
-          <para>the configuration file</para>
-        </listitem>
-      </varlistentry>
-      <varlistentry>
-        <term>/etc/pam.d</term>
-        <listitem>
-          <para>
-            the <emphasis remap="B">Linux-PAM</emphasis> configuration
-            directory. Generally, if this directory is present, the
-            <filename>/etc/pam.conf</filename> file is ignored.
-          </para>
-        </listitem>
-      </varlistentry>
-      <varlistentry>
-        <term>/usr/lib/pam.d</term>
-        <listitem>
-          <para>
-            the <emphasis remap="B">Linux-PAM</emphasis> vendor configuration
-            directory. Files in <filename>/etc/pam.d</filename> override
-            files with the same name in this directory.
-          </para>
-        </listitem>
-      </varlistentry>
-      <varlistentry condition="with_vendordir">
-        <term>%vendordir%/pam.d</term>
-        <listitem>
-          <para>
-	    additional <emphasis remap="B">Linux-PAM</emphasis> vendor
-	    configuration directory. Files in <filename>/etc/pam.d</filename>
-	    and <filename>/usr/lib/pam.d</filename> override files with the
-	    same name in this directory.
-          </para>
-        </listitem>
-      </varlistentry>
-    </variablelist>
-  </refsect1>
-
-  <refsect1 xml:id="pam8-errors">
-    <title>ERRORS</title>
-    <para>
-      Typically errors generated by the
-      <emphasis remap="B">Linux-PAM</emphasis> system of libraries, will
-      be written to <citerefentry>
-      <refentrytitle>syslog</refentrytitle><manvolnum>3</manvolnum>
-      </citerefentry>.
-    </para>
-  </refsect1>
-
-  <refsect1 xml:id="pam8-conforming_to">
-    <title>CONFORMING TO</title>
-    <para>
-      DCE-RFC 86.0, October 1995.
-      Contains additional features, but remains backwardly compatible
-      with this RFC.
-    </para>
-  </refsect1>
-
-  <refsect1 xml:id="pam8-see_also">
-    <title>SEE ALSO</title>
-    <para>
-      <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>3</manvolnum>
-      </citerefentry>,
-      <citerefentry>
-        <refentrytitle>pam_authenticate</refentrytitle><manvolnum>3</manvolnum>
-      </citerefentry>,
-      <citerefentry>
-        <refentrytitle>pam_sm_setcred</refentrytitle><manvolnum>3</manvolnum>
-      </citerefentry>,
-      <citerefentry>
-        <refentrytitle>pam_strerror</refentrytitle><manvolnum>3</manvolnum>
-      </citerefentry>,
-      <citerefentry>
-        <refentrytitle>PAM</refentrytitle><manvolnum>8</manvolnum>
-      </citerefentry>
-    </para>
-  </refsect1>
-</refentry>
diff --git a/doc/man/pam_acct_mgmt.3.xml b/doc/man/pam_acct_mgmt.3.xml
index de6a94a..6ff3ccb 100644
--- a/doc/man/pam_acct_mgmt.3.xml
+++ b/doc/man/pam_acct_mgmt.3.xml
@@ -136,7 +136,7 @@
         <refentrytitle>pam_strerror</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_authenticate.3.xml b/doc/man/pam_authenticate.3.xml
index 794a5c7..948b950 100644
--- a/doc/man/pam_authenticate.3.xml
+++ b/doc/man/pam_authenticate.3.xml
@@ -160,7 +160,7 @@
         <refentrytitle>pam_strerror</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_chauthtok.3.xml b/doc/man/pam_chauthtok.3.xml
index e184f45..95af359 100644
--- a/doc/man/pam_chauthtok.3.xml
+++ b/doc/man/pam_chauthtok.3.xml
@@ -155,7 +155,7 @@
         <refentrytitle>pam_strerror</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_conv.3.xml b/doc/man/pam_conv.3.xml
index 31834f3..96bfd23 100644
--- a/doc/man/pam_conv.3.xml
+++ b/doc/man/pam_conv.3.xml
@@ -219,7 +219,7 @@ struct pam_conv {
         <refentrytitle>pam_strerror</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_error.3.xml b/doc/man/pam_error.3.xml
index 0f294c2..82ea709 100644
--- a/doc/man/pam_error.3.xml
+++ b/doc/man/pam_error.3.xml
@@ -102,7 +102,7 @@
         <refentrytitle>pam_vprompt</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_get_authtok.3.xml b/doc/man/pam_get_authtok.3.xml
index 60e0a45..bfb04c1 100644
--- a/doc/man/pam_get_authtok.3.xml
+++ b/doc/man/pam_get_authtok.3.xml
@@ -230,7 +230,7 @@
     <title>SEE ALSO</title>
     <para>
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_getenv.3.xml b/doc/man/pam_getenv.3.xml
index df25863..b5dbc12 100644
--- a/doc/man/pam_getenv.3.xml
+++ b/doc/man/pam_getenv.3.xml
@@ -58,7 +58,7 @@
         <refentrytitle>pam_putenv</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_getenvlist.3.xml b/doc/man/pam_getenvlist.3.xml
index 54b1f41..7f755e5 100644
--- a/doc/man/pam_getenvlist.3.xml
+++ b/doc/man/pam_getenvlist.3.xml
@@ -76,7 +76,7 @@
         <refentrytitle>pam_putenv</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_info.3.xml b/doc/man/pam_info.3.xml
index 5155d41..9b4a3f0 100644
--- a/doc/man/pam_info.3.xml
+++ b/doc/man/pam_info.3.xml
@@ -90,7 +90,7 @@
     <title>SEE ALSO</title>
     <para>
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_misc_drop_env.3.xml b/doc/man/pam_misc_drop_env.3.xml
index a7f6cc8..c7a2576 100644
--- a/doc/man/pam_misc_drop_env.3.xml
+++ b/doc/man/pam_misc_drop_env.3.xml
@@ -43,7 +43,7 @@
         <refentrytitle>pam_getenvlist</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_misc_paste_env.3.xml b/doc/man/pam_misc_paste_env.3.xml
index 06194a9..2d99a1f 100644
--- a/doc/man/pam_misc_paste_env.3.xml
+++ b/doc/man/pam_misc_paste_env.3.xml
@@ -41,7 +41,7 @@
         <refentrytitle>pam_putenv</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_misc_setenv.3.xml b/doc/man/pam_misc_setenv.3.xml
index 4414d54..c9403c5 100644
--- a/doc/man/pam_misc_setenv.3.xml
+++ b/doc/man/pam_misc_setenv.3.xml
@@ -48,7 +48,7 @@
         <refentrytitle>pam_putenv</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_prompt.3.xml b/doc/man/pam_prompt.3.xml
index c65a0c9..b53f502 100644
--- a/doc/man/pam_prompt.3.xml
+++ b/doc/man/pam_prompt.3.xml
@@ -92,7 +92,7 @@
     <title>SEE ALSO</title>
     <para>
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>,
       <citerefentry>
         <refentrytitle>pam_conv</refentrytitle><manvolnum>3</manvolnum>
diff --git a/doc/man/pam_putenv.3.xml b/doc/man/pam_putenv.3.xml
index 7267046..8daca00 100644
--- a/doc/man/pam_putenv.3.xml
+++ b/doc/man/pam_putenv.3.xml
@@ -143,7 +143,7 @@
         <refentrytitle>pam_strerror</refentrytitle><manvolnum>3</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_strerror.3.xml b/doc/man/pam_strerror.3.xml
index b76cbc4..2c7a8a9 100644
--- a/doc/man/pam_strerror.3.xml
+++ b/doc/man/pam_strerror.3.xml
@@ -48,7 +48,7 @@
     <title>SEE ALSO</title>
     <para>
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/pam_syslog.3.xml b/doc/man/pam_syslog.3.xml
index f5be287..5005476 100644
--- a/doc/man/pam_syslog.3.xml
+++ b/doc/man/pam_syslog.3.xml
@@ -63,7 +63,7 @@
     <title>SEE ALSO</title>
     <para>
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_access/access.conf.5.xml b/modules/pam_access/access.conf.5.xml
index 0b93db0..65c6b69 100644
--- a/modules/pam_access/access.conf.5.xml
+++ b/modules/pam_access/access.conf.5.xml
@@ -240,7 +240,7 @@
     <para>
       <citerefentry><refentrytitle>pam_access</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum></citerefentry>
     </para>
   </refsect1>
 
diff --git a/modules/pam_access/pam_access.8.xml b/modules/pam_access/pam_access.8.xml
index c991d7a..dcc5039 100644
--- a/modules/pam_access/pam_access.8.xml
+++ b/modules/pam_access/pam_access.8.xml
@@ -285,7 +285,7 @@
         <refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>.
     </para>
   </refsect1>
diff --git a/modules/pam_debug/pam_debug.8.xml b/modules/pam_debug/pam_debug.8.xml
index 1c98f17..939c19b 100644
--- a/modules/pam_debug/pam_debug.8.xml
+++ b/modules/pam_debug/pam_debug.8.xml
@@ -213,7 +213,7 @@ auth    sufficient      pam_debug.so auth=success cred=success
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_deny/pam_deny.8.xml b/modules/pam_deny/pam_deny.8.xml
index db8fcb6..de41a59 100644
--- a/modules/pam_deny/pam_deny.8.xml
+++ b/modules/pam_deny/pam_deny.8.xml
@@ -117,7 +117,7 @@ other session  required       pam_deny.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_echo/pam_echo.8.xml b/modules/pam_echo/pam_echo.8.xml
index 07b793d..cf2d006 100644
--- a/modules/pam_echo/pam_echo.8.xml
+++ b/modules/pam_echo/pam_echo.8.xml
@@ -156,7 +156,7 @@ password required pam_unix.so
       <refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
     </citerefentry>,
     <citerefentry>
-      <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+      <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
     </citerefentry></para>
   </refsect1>
 
diff --git a/modules/pam_env/pam_env.8.xml b/modules/pam_env/pam_env.8.xml
index c7889e0..082c1d5 100644
--- a/modules/pam_env/pam_env.8.xml
+++ b/modules/pam_env/pam_env.8.xml
@@ -312,7 +312,7 @@
         <refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>,
       <citerefentry>
         <refentrytitle>environ</refentrytitle><manvolnum>7</manvolnum>
diff --git a/modules/pam_env/pam_env.conf.5.xml b/modules/pam_env/pam_env.conf.5.xml
index 46df480..da74046 100644
--- a/modules/pam_env/pam_env.conf.5.xml
+++ b/modules/pam_env/pam_env.conf.5.xml
@@ -135,7 +135,7 @@
     <para>
       <citerefentry><refentrytitle>pam_env</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
+      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>environ</refentrytitle><manvolnum>7</manvolnum></citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_exec/pam_exec.8.xml b/modules/pam_exec/pam_exec.8.xml
index 677d598..00b4b77 100644
--- a/modules/pam_exec/pam_exec.8.xml
+++ b/modules/pam_exec/pam_exec.8.xml
@@ -300,7 +300,7 @@
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_faildelay/pam_faildelay.8.xml b/modules/pam_faildelay/pam_faildelay.8.xml
index c31b507..49ec46f 100644
--- a/modules/pam_faildelay/pam_faildelay.8.xml
+++ b/modules/pam_faildelay/pam_faildelay.8.xml
@@ -118,7 +118,7 @@ auth  optional  pam_faildelay.so  delay=10000000
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_filter/pam_filter.8.xml b/modules/pam_filter/pam_filter.8.xml
index 8015f41..0b85e82 100644
--- a/modules/pam_filter/pam_filter.8.xml
+++ b/modules/pam_filter/pam_filter.8.xml
@@ -243,7 +243,7 @@
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_ftp/pam_ftp.8.xml b/modules/pam_ftp/pam_ftp.8.xml
index 03f3678..90079d3 100644
--- a/modules/pam_ftp/pam_ftp.8.xml
+++ b/modules/pam_ftp/pam_ftp.8.xml
@@ -165,7 +165,7 @@ auth    required    pam_listfile.so \
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_group/group.conf.5.xml b/modules/pam_group/group.conf.5.xml
index a8875b3..8d5b2d4 100644
--- a/modules/pam_group/group.conf.5.xml
+++ b/modules/pam_group/group.conf.5.xml
@@ -131,7 +131,7 @@ xsh; tty* ;%admin;Al0000-2400;plugdev
     <para>
       <citerefentry><refentrytitle>pam_group</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum></citerefentry>
     </para>
   </refsect1>
 
diff --git a/modules/pam_group/pam_group.8.xml b/modules/pam_group/pam_group.8.xml
index 695a7ba..292ee1c 100644
--- a/modules/pam_group/pam_group.8.xml
+++ b/modules/pam_group/pam_group.8.xml
@@ -149,7 +149,7 @@
         <refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>.
     </para>
   </refsect1>
diff --git a/modules/pam_issue/pam_issue.8.xml b/modules/pam_issue/pam_issue.8.xml
index 20d3245..02b31f6 100644
--- a/modules/pam_issue/pam_issue.8.xml
+++ b/modules/pam_issue/pam_issue.8.xml
@@ -216,7 +216,7 @@
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_keyinit/pam_keyinit.8.xml b/modules/pam_keyinit/pam_keyinit.8.xml
index 7b0a73b..0bab086 100644
--- a/modules/pam_keyinit/pam_keyinit.8.xml
+++ b/modules/pam_keyinit/pam_keyinit.8.xml
@@ -229,7 +229,7 @@ session  required  pam_keyinit.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>,
       <citerefentry>
 	<refentrytitle>keyctl</refentrytitle><manvolnum>1</manvolnum>
diff --git a/modules/pam_lastlog/pam_lastlog.8.xml b/modules/pam_lastlog/pam_lastlog.8.xml
index d990978..d641387 100644
--- a/modules/pam_lastlog/pam_lastlog.8.xml
+++ b/modules/pam_lastlog/pam_lastlog.8.xml
@@ -322,7 +322,7 @@
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_limits/limits.conf.5.xml b/modules/pam_limits/limits.conf.5.xml
index 348758a..652e37a 100644
--- a/modules/pam_limits/limits.conf.5.xml
+++ b/modules/pam_limits/limits.conf.5.xml
@@ -350,7 +350,7 @@ ftp             hard    nproc           0
     <para>
       <citerefentry><refentrytitle>pam_limits</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
+      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>getrlimit</refentrytitle><manvolnum>2</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>getrlimit</refentrytitle><manvolnum>3p</manvolnum></citerefentry>
     </para>
diff --git a/modules/pam_limits/pam_limits.8.xml b/modules/pam_limits/pam_limits.8.xml
index cca046c..8f026f0 100644
--- a/modules/pam_limits/pam_limits.8.xml
+++ b/modules/pam_limits/pam_limits.8.xml
@@ -264,7 +264,7 @@ session  required  pam_limits.so
         <refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>.
     </para>
   </refsect1>
diff --git a/modules/pam_listfile/pam_listfile.8.xml b/modules/pam_listfile/pam_listfile.8.xml
index 40a553d..d74fee1 100644
--- a/modules/pam_listfile/pam_listfile.8.xml
+++ b/modules/pam_listfile/pam_listfile.8.xml
@@ -278,7 +278,7 @@ auth    required       pam_listfile.so \
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_localuser/pam_localuser.8.xml b/modules/pam_localuser/pam_localuser.8.xml
index 2002d1d..e4b9e07 100644
--- a/modules/pam_localuser/pam_localuser.8.xml
+++ b/modules/pam_localuser/pam_localuser.8.xml
@@ -184,7 +184,7 @@ account required pam_wheel.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_loginuid/pam_loginuid.8.xml b/modules/pam_loginuid/pam_loginuid.8.xml
index d5285f0..1beba98 100644
--- a/modules/pam_loginuid/pam_loginuid.8.xml
+++ b/modules/pam_loginuid/pam_loginuid.8.xml
@@ -118,7 +118,7 @@ session    required     pam_loginuid.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>,
       <citerefentry>
 	<refentrytitle>auditctl</refentrytitle><manvolnum>8</manvolnum>
diff --git a/modules/pam_mail/pam_mail.8.xml b/modules/pam_mail/pam_mail.8.xml
index 2c0c054..9b4ce36 100644
--- a/modules/pam_mail/pam_mail.8.xml
+++ b/modules/pam_mail/pam_mail.8.xml
@@ -262,7 +262,7 @@ session  optional  pam_mail.so standard
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_mkhomedir/pam_mkhomedir.8.xml b/modules/pam_mkhomedir/pam_mkhomedir.8.xml
index ad95724..25f5497 100644
--- a/modules/pam_mkhomedir/pam_mkhomedir.8.xml
+++ b/modules/pam_mkhomedir/pam_mkhomedir.8.xml
@@ -202,7 +202,7 @@
         <refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>.
     </para>
   </refsect1>
diff --git a/modules/pam_motd/pam_motd.8.xml b/modules/pam_motd/pam_motd.8.xml
index 7442037..2fc5310 100644
--- a/modules/pam_motd/pam_motd.8.xml
+++ b/modules/pam_motd/pam_motd.8.xml
@@ -193,7 +193,7 @@ session  optional  pam_motd.so motd=/elsewhere/motd motd_dir=/elsewhere/motd.d
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_namespace/namespace.conf.5.xml b/modules/pam_namespace/namespace.conf.5.xml
index 54f9431..15aef5c 100644
--- a/modules/pam_namespace/namespace.conf.5.xml
+++ b/modules/pam_namespace/namespace.conf.5.xml
@@ -226,7 +226,7 @@
     <para>
       <citerefentry><refentrytitle>pam_namespace</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum></citerefentry>
     </para>
   </refsect1>
 
diff --git a/modules/pam_namespace/pam_namespace.8.xml b/modules/pam_namespace/pam_namespace.8.xml
index a866d2e..0896372 100644
--- a/modules/pam_namespace/pam_namespace.8.xml
+++ b/modules/pam_namespace/pam_namespace.8.xml
@@ -392,7 +392,7 @@
         <refentrytitle>mount</refentrytitle><manvolnum>8</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>.
     </para>
   </refsect1>
diff --git a/modules/pam_nologin/pam_nologin.8.xml b/modules/pam_nologin/pam_nologin.8.xml
index 1ea725c..1cc721a 100644
--- a/modules/pam_nologin/pam_nologin.8.xml
+++ b/modules/pam_nologin/pam_nologin.8.xml
@@ -157,7 +157,7 @@ auth  required  pam_nologin.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_permit/pam_permit.8.xml b/modules/pam_permit/pam_permit.8.xml
index 0634e5e..9e6c7d0 100644
--- a/modules/pam_permit/pam_permit.8.xml
+++ b/modules/pam_permit/pam_permit.8.xml
@@ -88,7 +88,7 @@ account  required  pam_permit.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_pwhistory/pam_pwhistory.8.xml b/modules/pam_pwhistory/pam_pwhistory.8.xml
index d83d8d9..a5185fc 100644
--- a/modules/pam_pwhistory/pam_pwhistory.8.xml
+++ b/modules/pam_pwhistory/pam_pwhistory.8.xml
@@ -282,7 +282,7 @@ password     required       pam_unix.so        use_authtok
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
       <citerefentry>
 	<refentrytitle>pam_get_authtok</refentrytitle><manvolnum>3</manvolnum>
diff --git a/modules/pam_rhosts/pam_rhosts.8.xml b/modules/pam_rhosts/pam_rhosts.8.xml
index b8a5c1c..41d541c 100644
--- a/modules/pam_rhosts/pam_rhosts.8.xml
+++ b/modules/pam_rhosts/pam_rhosts.8.xml
@@ -153,7 +153,7 @@ auth     required       pam_unix.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_rootok/pam_rootok.8.xml b/modules/pam_rootok/pam_rootok.8.xml
index a79c073..f30ad37 100644
--- a/modules/pam_rootok/pam_rootok.8.xml
+++ b/modules/pam_rootok/pam_rootok.8.xml
@@ -113,7 +113,7 @@ auth  required     pam_unix.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_securetty/pam_securetty.8.xml b/modules/pam_securetty/pam_securetty.8.xml
index 9038f5b..fcf0e88 100644
--- a/modules/pam_securetty/pam_securetty.8.xml
+++ b/modules/pam_securetty/pam_securetty.8.xml
@@ -184,7 +184,7 @@ auth  required  pam_unix.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_selinux/pam_selinux.8.xml b/modules/pam_selinux/pam_selinux.8.xml
index 3aa632c..7ec5daf 100644
--- a/modules/pam_selinux/pam_selinux.8.xml
+++ b/modules/pam_selinux/pam_selinux.8.xml
@@ -255,7 +255,7 @@ session  optional  pam_selinux.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>,
       <citerefentry>
 	<refentrytitle>selinux</refentrytitle><manvolnum>8</manvolnum>
diff --git a/modules/pam_sepermit/pam_sepermit.8.xml b/modules/pam_sepermit/pam_sepermit.8.xml
index 114864a..9efb204 100644
--- a/modules/pam_sepermit/pam_sepermit.8.xml
+++ b/modules/pam_sepermit/pam_sepermit.8.xml
@@ -177,7 +177,7 @@ session  required  pam_permit.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
       <citerefentry>
 	<refentrytitle>selinux</refentrytitle><manvolnum>8</manvolnum>
diff --git a/modules/pam_sepermit/sepermit.conf.5.xml b/modules/pam_sepermit/sepermit.conf.5.xml
index ff924ce..1f1dcae 100644
--- a/modules/pam_sepermit/sepermit.conf.5.xml
+++ b/modules/pam_sepermit/sepermit.conf.5.xml
@@ -93,7 +93,7 @@
     <para>
       <citerefentry><refentrytitle>pam_sepermit</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
+      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>selinux</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
     </para>
   </refsect1>
diff --git a/modules/pam_shells/pam_shells.8.xml b/modules/pam_shells/pam_shells.8.xml
index e1b35a3..7889702 100644
--- a/modules/pam_shells/pam_shells.8.xml
+++ b/modules/pam_shells/pam_shells.8.xml
@@ -115,7 +115,7 @@ auth  required  pam_shells.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_succeed_if/pam_succeed_if.8.xml b/modules/pam_succeed_if/pam_succeed_if.8.xml
index 90fd114..b8f65e7 100644
--- a/modules/pam_succeed_if/pam_succeed_if.8.xml
+++ b/modules/pam_succeed_if/pam_succeed_if.8.xml
@@ -291,7 +291,7 @@ type required othermodule.so arguments...
         <refentrytitle>glob</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_time/pam_time.8.xml b/modules/pam_time/pam_time.8.xml
index 1fa60a1..748bcd1 100644
--- a/modules/pam_time/pam_time.8.xml
+++ b/modules/pam_time/pam_time.8.xml
@@ -186,7 +186,7 @@ login  account  required  pam_time.so
         <refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-        <refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+        <refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>.
     </para>
   </refsect1>
diff --git a/modules/pam_time/time.conf.5.xml b/modules/pam_time/time.conf.5.xml
index 3fe263d..30c9a92 100644
--- a/modules/pam_time/time.conf.5.xml
+++ b/modules/pam_time/time.conf.5.xml
@@ -133,7 +133,7 @@ games ; * ; !waster ; Wd0000-2400 | Wk1800-0800
     <para>
       <citerefentry><refentrytitle>pam_time</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+      <citerefentry><refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum></citerefentry>
     </para>
   </refsect1>
 
diff --git a/modules/pam_timestamp/pam_timestamp.8.xml b/modules/pam_timestamp/pam_timestamp.8.xml
index a763ad8..e6b2df7 100644
--- a/modules/pam_timestamp/pam_timestamp.8.xml
+++ b/modules/pam_timestamp/pam_timestamp.8.xml
@@ -190,7 +190,7 @@ session optional pam_timestamp.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_timestamp/pam_timestamp_check.8.xml b/modules/pam_timestamp/pam_timestamp_check.8.xml
index 7f850ae..7a7770a 100644
--- a/modules/pam_timestamp/pam_timestamp_check.8.xml
+++ b/modules/pam_timestamp/pam_timestamp_check.8.xml
@@ -189,7 +189,7 @@ session optional pam_timestamp.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_tty_audit/pam_tty_audit.8.xml b/modules/pam_tty_audit/pam_tty_audit.8.xml
index b46bbf7..79d8115 100644
--- a/modules/pam_tty_audit/pam_tty_audit.8.xml
+++ b/modules/pam_tty_audit/pam_tty_audit.8.xml
@@ -178,7 +178,7 @@ session	required pam_tty_audit.so disable=* enable=root
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_umask/pam_umask.8.xml b/modules/pam_umask/pam_umask.8.xml
index d2cead4..54a44ef 100644
--- a/modules/pam_umask/pam_umask.8.xml
+++ b/modules/pam_umask/pam_umask.8.xml
@@ -245,7 +245,7 @@
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_unix/pam_unix.8.xml b/modules/pam_unix/pam_unix.8.xml
index d02320b..7e4fd5d 100644
--- a/modules/pam_unix/pam_unix.8.xml
+++ b/modules/pam_unix/pam_unix.8.xml
@@ -556,7 +556,7 @@ session    required   pam_unix.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_userdb/pam_userdb.8.xml b/modules/pam_userdb/pam_userdb.8.xml
index 0f96410..86ba895 100644
--- a/modules/pam_userdb/pam_userdb.8.xml
+++ b/modules/pam_userdb/pam_userdb.8.xml
@@ -276,7 +276,7 @@ auth  sufficient pam_userdb.so icase db=/etc/dbtest
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_warn/pam_warn.8.xml b/modules/pam_warn/pam_warn.8.xml
index a20c5f7..a69e1d6 100644
--- a/modules/pam_warn/pam_warn.8.xml
+++ b/modules/pam_warn/pam_warn.8.xml
@@ -87,7 +87,7 @@ other session  required       pam_deny.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_wheel/pam_wheel.8.xml b/modules/pam_wheel/pam_wheel.8.xml
index b42e27d..86f2828 100644
--- a/modules/pam_wheel/pam_wheel.8.xml
+++ b/modules/pam_wheel/pam_wheel.8.xml
@@ -210,7 +210,7 @@ su      auth     required       pam_unix.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
diff --git a/modules/pam_xauth/pam_xauth.8.xml b/modules/pam_xauth/pam_xauth.8.xml
index f5fc5a3..214226b 100644
--- a/modules/pam_xauth/pam_xauth.8.xml
+++ b/modules/pam_xauth/pam_xauth.8.xml
@@ -273,7 +273,7 @@ session  optional  pam_xauth.so
 	<refentrytitle>pam.d</refentrytitle><manvolnum>5</manvolnum>
       </citerefentry>,
       <citerefentry>
-	<refentrytitle>pam</refentrytitle><manvolnum>8</manvolnum>
+	<refentrytitle>pam</refentrytitle><manvolnum>7</manvolnum>
       </citerefentry>
     </para>
   </refsect1>
